### PR TITLE
Add ruby 3.2 to CI matrix

### PR DESCRIPTION
**Pull request by pariser**

### CODE REVIEW

File: sparsam.gemspec [hunk 1]
```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |s|
 
   s.require_paths = %w(lib ext)
 
-  s.add_development_dependency 'rspec', '~> 2.10.0'
+  s.add_development_dependency 'rspec'
```
Comment (pariser) [inline sparsam.gemspec:26]: now running rspec 3

**Review by danielevans — APPROVED** [review]

File: .github/workflows/rspec_rubocop.yml [hunk 2]
```diff
@@ -36,6 +36,8 @@ jobs:
           bundler-version: latest
         - ruby-version: 3.1
           bundler-version: latest
+        - ruby-version: 3.2
```
Comment (shepting) [inline .github/workflows/rspec_rubocop.yml:39]: 👍

File: sparsam.gemspec [hunk 3]
```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |s|
...
   s.add_development_dependency "bundler"
-  s.add_development_dependency 'rake', '< 11.0'
```
Comment (shepting) [inline sparsam.gemspec:28]: 💯

**Review by Umofomia — APPROVED** [review]
Looks good. You're adding support for Ruby 3.2 to sparsam by adding it to the test matrix and removing its version constraints on `rspec` and `rake`.

File: sparsam.gemspec [hunk 4]
```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |s|
 
   s.require_paths = %w(lib ext)
 
-  s.add_development_dependency 'rspec', '~> 2.10.0'
```
Comment (shepting) [inline sparsam.gemspec:26]: ❤️

**Pull request merged**